pennylane/numpy/tensor.py

```diff
@@ -113,6 +113,43 @@ def __array_wrap__(self, obj):
         out_arr = tensor(obj, requires_grad=self.requires_grad)
         return super().__array_wrap__(out_arr)
 
+    def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
+        # pylint: disable=no-member,attribute-defined-outside-init
+        outputs = [
+            i.view(onp.ndarray) if hasattr(i, "unwrap") else i for i in kwargs.get("out", ())

```

Comment:
What would the `hasattr` distinguish here?

Reply:
You're right, I think we can simply do
```python
outputs = [i.view(onp.ndarray) for i in kwargs.get("out", ())]
```
I'll change it and see if the tests pass.

Reply:
it works!